client/js/app/utils/ExplorerUtils.js

```diff
@@ -79,36 +82,34 @@ module.exports = {
     }
     var params = _.cloneDeep(explorer.query);
 
-    // Set the timeframe (will get removed if it's null o undefined)
-    params.timeframe = module.exports.getTimeframe(explorer);
```

Comment:
Whoops, meant for this to be down at the `TimeframeUtils.getTimeParameters` line. (line `85`).